cmd/linuxdb.yaml

```diff
@@ -307,3 +307,22 @@ distributions:
           gccversioncommand: *gccdumpfullversion
           programs: *opensusedefaultprograms
           libraries: *opensusedefaultlibraries
+  crux:
+      id: crux
+      releases:
+        default:
+          version: default
+          name: Crux Linux
+          gccversioncommand: *gccdumpversion
+          programs:
+            - name: gcc
+              help: Please install with `sudo prt-get depinst gcc-c++ make` and try again
+            - name: pkg-config
+              help: Please install with `sudo prt-get depinst pkgconf-pkg-config` and try again
```

Comment:
Just checking pkgconf-pkg-config is right

Reply:
Thanks for doing this! Will make sure `prt` gets added to v2!

Reply:
typo on my end, commited the fix, thanks for catching it